src/Meilisearch/SearchResult.cs

```diff
@@ -79,5 +81,9 @@ public class SearchResult<T> : ISearchable<T>
         /// </summary>
         [JsonPropertyName("facetStats")]
         public IReadOnlyDictionary<string, FacetStat> FacetStats { get; }
+
+        /// <inheritdoc/>
```

Comment:
Same than for the first one

Reply:
https://github.com/meilisearch/meilisearch-dotnet/pull/416/commits/dc6c9b6c8769b2e297a43fa91ab6bdc23119a8df